src/MobileNavbar/MobileNavbar.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MobileNavbar } from './MobileNavbar';
+import { BrowserRouter } from 'react-router-dom';
+
+test('renders the MobileNavbar component without errors', () => {
```

Comment:
Could we add a test that clicks the button and checks that the MobileNavBar is now showing please